configure.in

```diff
@@ -164,7 +164,9 @@ CITUSAC_PROG_CC_CFLAGS_OPT([-Wmissing-format-attribute])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wmissing-declarations])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wmissing-prototypes])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wshadow])
-CITUSAC_PROG_CC_CFLAGS_OPT([-Werror=vla])  # visual studio does not support these
+CITUSAC_PROG_CC_CFLAGS_OPT([-Wvla])  # visual studio does not support these
+CITUSAC_PROG_CC_CFLAGS_OPT([-Werror])
+CITUSAC_PROG_CC_CFLAGS_OPT([-Wfatal-errors])
```

Comment:
> Included `Wfatal-errors` to avoid offending warnings being scrolled off screen

Uh… do you not have a scrollback buffer? I _definitely_ do not want this behavior. In my development environment, I compile, and all errors get marked in my text buffers, based on scanning the build output. If the build halted after the first error, it would increase the cycle time substantially, since I'd need to trigger a new build between addressing each error (this wouldn't just apply to warnings, but also actual errors during e.g. developer work).

Even if we add `error` to CI, I _don't_ think this should be added anywhere (if you want it locally, sure, configure your development environment with this flag. I had to look up what it did, frankly).